hack/mdtoc-version

```diff
@@ -0,0 +1 @@
+ee652eb78c047a7b6c7417d9324a97bb05689563
```

Comment:
Where's the table of contents this file refers to?

Reply:
This was used in the toc check which I didn't add this PR. Discussed with Zhengsheng offline and this toc check is not a must-have and didn't cover the flow-visibility doc before. So I removed this file for now.